cypress/e2e/marc/marc-authority/edit-authority-record/verify-source-value-displays-only-last-name.cy.js

```diff
@@ -0,0 +1,111 @@
+        JobProfiles.search(testData.marcFile.jobProfileToRun);
+        JobProfiles.runImportFile();
+        JobProfiles.waitFileIsImported(testData.marcFile.fileName);
+        Logs.checkStatusOfJobProfile('Completed');
+        Logs.openFileDetails(testData.marcFile.fileName);
+        Logs.getCreatedItemsID(0).then((link) => {
+          testData.createdAuthorityIDs.push(link.split('/')[5]);
+        });
+      });
+
+      cy.createTempUser([
+        Permissions.uiMarcAuthoritiesAuthorityRecordEdit.gui,
+        Permissions.uiMarcAuthoritiesAuthorityRecordView.gui,
+        Permissions.uiQuickMarcQuickMarcAuthoritiesEditorAll.gui,
+      ]).then((createdUserProperties) => {
+        testData.userProperties = createdUserProperties;
+        cy.login(testData.userProperties.username, testData.userProperties.password, {
+          path: TopMenu.marcAuthorities,
+          waiter: MarcAuthorities.waitLoading,
+        });
+      });
+    });
+
+    after('Delete test data', () => {
+      cy.getAdminToken();
+      Users.deleteViaApi(testData.userProperties.userId);
+      testData.createdAuthorityIDs.forEach((id) => {
+        MarcAuthority.deleteViaAPI(id);
+      });
+    });
+
+    it(
+      'C356848 Verify that "Source" value displays only the Last name of user, which edited record, when First name of user is not populated (spitfire) (TaaS)',
+      { tags: ['extendedPath', 'spitfire'] },
+      () => {
+        // Step 1: Fill in the input field at "Search & filter" pane with the search query
+        MarcAuthorities.searchByParameter(
+          testData.authority.searchOption,
+          testData.authority.searchInput,
+        );
+        MarcAuthorities.checkAfterSearch('Authorized', testData.authority.searchInput);
+
+        // Step 2,3: View any "MARC Authority" record by clicking on its title
+        MarcAuthorities.selectItem(testData.authority.searchInput, false);
+
+        // Step 4: Click on the "Actions" button and select the "Edit" option
+        MarcAuthority.edit();
+        QuickMarcEditor.waitLoading();
+        // Step 5: Verify that "Source" value does not match with your user's name
+        QuickMarcEditor.checkUserNameInHeader(testData.source.lastName, testData.source.firstName);
+
+        // Step 6: Edit $a subfield of any 1XX MARC tag
+        QuickMarcEditor.updateExistingField('100', '$a New Content');
+
+        // Step 7: Click on the "Save & close" button
+        QuickMarcEditor.pressSaveAndClose();
+        QuickMarcEditor.checkCallout(testData.calloutMessage);
+        MarcAuthority.waitLoading();
+
+        // Step 8: Click on the "Actions" button and select the "Edit" option
+        MarcAuthority.edit();
+        QuickMarcEditor.waitLoading();
+
+        // Step 9: Verify that "Source" value matches with your user's Last name
+        QuickMarcEditor.checkUserNameInHeader(
+          testData.userProperties.firstName,
+          testData.userProperties.lastName,
```

Comment:
Could you please update the test so that:
- test user would only have Last name and no First name specified (probably would require additional/updated methods for user creation)
- check that only Last name of the user is then shown in header
This check is described in the test case and is the main idea of it